pkg/tfgen/generate_schema_test.go

```diff
@@ -189,7 +189,7 @@ func TestNestedDescriptions(t *testing.T) {
 	schema, err := GenerateSchema(provider, diag.DefaultSink(io.Discard, io.Discard, diag.FormatOptions{
 		Color: colors.Never,
 	}))
-	assert.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
Is there a reason to prefer `require`?

Reply:
Yes. `require` stops the test when it errors, while `assert` just fails the test. 

If `err != nil`, then we don't want to assert that the JSON is equal (we know it won't be), so we should stop the test (with `require`).